<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarketReportSetupDialog</class>
 <widget class="QDialog" name="MarketReportSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>463</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Setup Market Report</string>
  </property>
  <widget class="Q3GroupBox" name="groupBox22">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Setup Market Report</string>
   </property>
   <widget class="Q3Table" name="sortTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>180</height>
     </rect>
    </property>
    <property name="numRows">
     <number>6</number>
    </property>
    <property name="numCols">
     <number>2</number>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Sort By</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descending?</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>359</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acceptButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>359</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupByWeightGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>311</width>
      <height>136</height>
     </rect>
    </property>
    <property name="title">
     <string>Group by weight</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Starting weight</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="startWeightEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ending weight</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="sizeEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="endWeightEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Size of class  </string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sortTable</tabstop>
  <tabstop>groupByWeightGroupBox</tabstop>
  <tabstop>startWeightEdit</tabstop>
  <tabstop>endWeightEdit</tabstop>
  <tabstop>sizeEdit</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>acceptButton</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <includes>
  <include location="local">dao/setupdao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
